nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -38,6 +38,8 @@
   Users can use it by `services.displayManager.ly.enable` and config it by
   `services.displayManager.ly.settings` to generate `/etc/ly/config.ini`
 
+- `srcOnly` was rewritten to be more readable, have additional warnings in the event that something is probably wrong, use the `stdenv` provided by the derivation, and Noogle-compatible documentation was added.
```

Comment:
I'd drop the release notes, but I don't care enough to insist. Based on what's above and below this note, this is in some different class than NixOS module updates.